packages/config/config/devices/0x0312/nzw30t.json

```
@@ -0,0 +1,54 @@
+// NIE Technology Co., Ltd. NZW30T
+// Inovelli In-Wall Switch (On/Off) Scene Enabled
+{
+	"manufacturer": "NIE Technology Co., Ltd.",
+	"manufacturerId": "0x0312",
+	"label": "NZW30T",
+	"description": "Inovelli In-Wall Switch (On/Off) Scene Enabled",
```

```suggestion
	"manufacturer": "Inovelli",
	"manufacturerId": "0x0312",
	"label": "NZW30T",
	"description": "In-Wall Switch (On/Off) Scene Enabled",
```
I guess that would be better and cause less confusion in the long run.

---

Makes sense, I have some Inovelli NZW30S and NZW31S. Should I wait to replace the configuration files? You guys have a lot going on right now so I figured to ask first.

---

Right now we have batched out new device files, so if it is a change to an existing device file you can go ahead.